### ISSUE TITLE
Validator vs. data use checks in IPT?

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#1394]
**Validator vs. data use checks in IPT?**

A publisher asks why do we have a validator, while when publishing through IPT includes pre-publication checks? A bit more clarity on the validator pages would help, it seems

-----

User provided contact info: @dschigel
System: Firefox 52.0.0 / Windows 10.0.0
User: [See in registry](https://www.gbif.org/api/feedback/user/b13eceb704212e5aa9849a33965b4cff:5554747e21e6edb38bd37c45dd610ca0dd4a61aade681a2a1801b0c426bc7c0a4a2fa773633916893aed1fc405a4165a0a80638cc2fbc5b6eac74caa524cb9a3)
Referer: https://www.gbif.org/tools/data-validator
Window size: width 1408 - height 679
[API log](http://elk.gbif.org:5601/app/kibana?#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2018-06-12T09:30:59.659Z',mode:absolute,to:'2018-06-12T09:36:59.659Z'))&_a=(columns:!(_source),index:'prod-varnish-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499')),sort:!('@timestamp',desc)))
[Site log](http://elk.gbif.org:5601/app/kibana?#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2018-06-12T09:30:59.659Z',mode:absolute,to:'2018-06-12T09:36:59.659Z'))&_a=(columns:!(_source),index:'prod-portal-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499')),sort:!('@timestamp',desc)))
System health at time of feedback: OPERATIONAL

**Comment by dschigel** [comment]
The question I got was more or less WHY use validator if you anyhow do the pre-publicaitons checks with IPT? I am sure there are benefits of validator-first approach, perhaps a Why? section could be added between What and How. Specifically what are the benefits of the validator report compared to publishing failure report in IPT.

**Comment by dnoesgaard** [comment]
Aside from the obvious difference that not everyone has access to an IPT, I actually don't know exactly. If someone could explain the difference to me, I'd be happy to write it up. 

@thomasstjerne, @fmendezh ?

**Comment by MattBlissett** [comment]
The IPT checks the structure of the data, and maybe some simple validity checks — I don't actually know offhand. (@jlegind might?)

The validator simulates the full ingestion process into GBIF's occurrence index, matching names against the GBIF backbone, running country/location checks and so on.

**Comment by dschigel** [comment]
Thanks all. My only point triggered by the question that if we know the benefits of the validator vs. IPT only checks, we might want expose these benefits a bit better at the validator page.

E.g.
"
Why use validator?
Publishing data may be blocked by the critical issues reported by the IPT. Other data issues do not prevent data publishing, but those non-critical issues are reported at the resulting dataset page and my reduce the suitability of the data for certain uses. Data validator offers an opportunity to... It is a useful to for... in the situations...etc."

**Comment by jlegind** [comment]
Since the Validator page does not have a 'motivation' paragraph, how about we add this?

- The new validator is consistent with what happens at indexing in GBIF — data said to be valid are indexable 

- It exposes information about data interpretation issues that later can appear while indexing a file in the GBIF portal — Indexing and the Validator use the same underlying libraries

- Unlike the old DwC validator, this one does meaningful content level validation — not only correctness of the file (e.g. referential integrity) but also testing scientific name "higher rank matching"

- Previews EML metadata

- Accepts DwC archives as well as Excel sheets and csv files

**Comment by jlegind** [comment]
Out of scope
